Library/Scenes/Books/BooksTableViewController.swift

```diff
@@ -73,13 +73,13 @@ class BooksTableViewController: UITableViewController, BooksView {
 	                        titleForDeleteConfirmationButtonForRowAt indexPath: IndexPath) -> String? {
 		return presenter.titleForDeleteButton(row: indexPath.row)
 	}
-	
-	override func tableView(_ tableView: UITableView,
-	                        commit editingStyle: UITableViewCellEditingStyle,
-	                        forRowAt indexPath: IndexPath) {
-		presenter.deleteButtonPressed(row: indexPath.row)
-	}
-	
+    
```

Comment:
Please use consistent indentation with the rest of the project (tabs instead of spaces).